src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Notification.java

```diff
@@ -1,7 +1,10 @@
 package edu.psu.sweng894.group1.capstone.data.entities;
 
```

Comment:
We need to save notifications for the DB as you need to associate a notification with a user and a single action can generate multiple notifications

Reply:
Still not a reason to save it to the DB... we can add a mailer and send a notification at that point? Again, premature!